### PR TITLE
Add working with different topologies example

### PR DESCRIPTION
**Pull request by arcondello**

### CODE REVIEW

File: examples/topology_samplers.rst [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+>>> from dwave.system import DWaveSampler, EmbeddingComposite
+
+Creating a Chimera Sampler
+--------------------------
+
+As detailed in :ref:`cpu`, you might want to use a classical solver while
+developing your code or writing tests. However, it is sometimes useful to
+work with a software solver that behaves more like a quantum computer.
+
+One of the key features of the quantum computer is the :term:`hardware graph`, which
```
Comment (JoelPasvolsky) [inline examples/topology_samplers.rst]: Unclear what you mean by this term: do you mean topology (Chimera, Pegasus) or are you referring to the "working graph" which defines that actual as opposed to the ideal topology graph?

File: examples/topology_samplers.rst [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+defines the connectivity allowed by the :term:`binary quadratic model`.
+
+To create a software solver with the same connectivity of a quantum computer
```
Comment (JoelPasvolsky) [inline examples/topology_samplers.rst]: same connectivity as a D-Wave 2000Q quantum computer

File: examples/topology_samplers.rst [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+:std:doc:`dwave_networkx <networkx:index>` and the
+:class:`neal.SimulatedAnnealingSampler` found in :std:doc:`dwave_neal <neal:index>`
+to construct a sampler.
+
+>>> P6 = dnx.pegasus_graph(6)
+>>> classical_sampler = neal.SimulatedAnnealingSampler()
+>>> sampler = dimod.StructureComposite(classical_sampler, P6.nodes, P6.edges)
+
+Similarly we can explore clique embeddings for a 40-variables fully connected
+problem with a 680 node Pegasus graph
```
Comment (JoelPasvolsky) [inline examples/topology_samplers.rst]: using?

File: examples/topology_samplers.rst [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+:func:`~dwave.embedding.chimera.find_clique_embedding` function to find the
+embedding and determine the maximum chain length.
+
+>>> num_variables = 40
+>>> embedding = dwave.embedding.chimera.find_clique_embedding(num_variables, 16)
+>>> max(len(chain) for chain in embedding.values())
+11
+
+
+Working with Pegasus
```
Comment (JoelPasvolsky) [inline examples/topology_samplers.rst]: Can you make this section parallel the Creating a Chimera Sampler, move the Working With Embeddings below both those sections, and maybe rename to something like "Embedding a K_40 Problem"?

File: glossary.rst [hunk 5]
```diff
@@ -11,11 +11,26 @@ Glossary
...
+         the source graph. See :term:`embedding`.
+
       Chimera
          The D-Wave :term:`QPU` is a lattice of interconnected qubits. While some qubits
          connect to others via couplers, the D-Wave QPU is not fully connected.
          Instead, the qubits interconnect in an architecture known as Chimera.
 
+      Complete graph
+      Fully connected
+          See `complete graph`_. A fully connected or complete
```
Comment (JoelPasvolsky) [inline glossary.rst]: See `complete graph`_. ? Did you mean to put the description under Complete Graph and the reference under Fully connect, placed in the alphabetically-ordered place?

File: glossary.rst [hunk 6]
```diff
@@ -56,6 +71,11 @@ Glossary
          a qubit :math:`q_i`, and :math:`h_i` and :math:`J_{i,j}` are the qubit
          biases and coupling strengths.
 
+      Hardware graph
```
Comment (JoelPasvolsky) [inline glossary.rst]: That reference has working graph

File: glossary.rst [hunk 7]
```diff
@@ -141,3 +161,11 @@ Glossary
...
          A resource that runs a problem. Some solvers interface to the :term:`QPU`;
          others leverage CPU and GPU resources.
+
+      Subgraph
+         See subgraph_.
+
+         .. _subgraph: https://en.wikipedia.org/wiki/Glossary_of_graph_theory_terms#subgraph
+
+      Working graph
+         In a D-Wave QPU, the set of qubits and couplers that are available for computation is known as the working graph. The yield of a working graph is typically less than the total number of qubits and couplers that are fabricated and physically present in the QPU.
```
Comment (JoelPasvolsky) [inline glossary.rst]: less than 100% of qubits and couplers that are fabricated

File: glossary.rst (hunk 5 above)
Comment (arcondello) [reply glossary.rst]: That links wikipedia

File: glossary.rst (hunk 7 above)
Comment (arcondello) [reply glossary.rst]: That one is copy and pasted from you! :smile:  I'll change it

**Pull request merged**